inspirehep/modules/migrator/tasks.py

```diff
@@ -176,7 +177,6 @@ def continuous_migration():
                         errors = "ValidationError: Record {0}: {1}".format(
                             recid, e
                         )
-                        current_app.logger.warning(errors)
```

Comment:
I didn't know how to test the continous migration. This should be replaced with ` migrate_and_insert_record(raw_record)` anyway